azext_edge/edge/providers/rpsaas/user_strings.py

```diff
@@ -11,7 +11,7 @@
     "The command may fail."
 CLUSTER_OFFLINE_MSG = "Cluster {0} is not connected. The cluster may not update correctly."
 MISSING_CLUSTER_CUSTOM_LOCATION_ERROR = "Need to provide either cluster name or custom location"
-MISSING_EXTENSION_ERROR = "Cluster {0} is missing the {1} extension."
+MISSING_EXTENSION_ERROR = "Cluster {0} is missing the {1} extension with enabled features."
```

Comment:
Should we show users which feature needs to be enabled to avoid the error?

Reply:
it was in the unsaved file in the latest commit rip